src/GeoStatsBase.jl

```diff
@@ -26,6 +26,7 @@ using ScientificTypes
 using LossFunctions
 using RecipesBase
 using Parameters
+using ReferenceFrameRotations: angle_to_dcm, quat_to_dcm, Quaternion
```

Comment:
Mind moving this line to right after `using NearestNeighbors`? Also, if you can move the `Quaternion` to the beginning before the function names, that would be great.

Reply:
Done. Removed quaternion functions (discussion in next commments)